### PR TITLE
[DOC] Initial commit of furo theme

### LINKED ISSUE #94: [DOC] Update the doc theme
The theme for the built docs is a bit clunky ; I'd suggest we update to [pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/install.html), following MNE and sklearn, or [furo](https://pradyunsg.me/furo/), following nilearn.

### PR DESCRIPTION
**Pull request by emdupre**
Closes #94 

- Adopts furo theme for sphinx docs
- Makes initial content changes for compatibility with new theme

### CODE REVIEW

**Comment by emdupre** [comment]
Not perfect, but I think it's an improvement !

I haven't significantly changed the content, since I don't want this PR to be any larger than necessary. I can open a different PR for that once this is in !

**Review by bthirion — APPROVED** [review]
The rendered doc LGTM.

**Review by pbarbarant — APPROVED** [review]
Caught one misslabeled file (see the comment). LGTM otherwise, awesome job 🙌!

File: doc/index.rst [hunk 1]
```diff
@@ -1,18 +1,148 @@
...
-    It leverages the `Nilearn <http://nilearn.github.io>`_ and `scikit-learn <http://scikit-learn.org>`_ toolboxes for alignment usecases such as new data prediction or improved `decoding <https://nilearn.github.io/decoding/index.html>`_.
 
 .. toctree::
-    :hidden:
+   :hidden:
+   :includehidden:
+   :titlesonly:
+
+   quickstart.md
+   auto_examples/reference.rst
```
[the remark this answers is not in the document]
Comment (emdupre) [reply doc/index.rst:135]: Great catch !! It's rendering correctly now : 

Screenshot 2024-11-04 at 11 35 31

**Comment by pbarbarant** [comment]
I think you can merge once the CI is done.

**Comment by emdupre** [comment]
Just turned green, so merging now ! 🚀

**Pull request merged**